### ISSUE TITLE
Newer build of A11 ROM Lenovo ThinkSmart View?

### ISSUE THREAD

**Issue by pgale** [dinki/View-Assist#2]
Anyone got a more stable version running yet?

**Comment by dinki** [comment]
What issues are you having in terms of stability.  I'm finding the A11 ROM linked in the forum to be very stable in terms of the device not crashing.  I do see two issues and both are seen at boot:

* Sometimes on boot the device  will crash to the Lenovo screen.  This seems remedied by pulling power for a few minutes and then booting back up.
* Sometimes on boot the device will make it to the desktop but will crash to Lenovo screen shortly after.  The workaround for this is to quickly turn wifi off when reaching the desktop and leaving it off for a short time and then reenabling.

Neither of these issues are great but short of those things are pretty good otherwise.

**Comment by pgale** [comment]
Thanks for the info.

I currently have five of these with four running the 8.1 ROM (mattmon) with fully kiosk. Works pretty well apart from no audio at all through the browser, even with alternate webviews loaded. So I've been looking around for something else. My latest device (No 5) is my test device so have been using that. I installed following your various links and managed to get it installed via Windows (a very long process of USB cables and a different PC in the end - QFIL or the drivers seems to be very picky over USB ports).

All good but then I just couldn't get the updated webview to work and after a lot of effort, gave up for a bit.

I've just tried again but using my Debian install on a Pi4 which has been fabulous. It installed OK but then just locks on the Lenovo logo. Have tried leaving it for a long time as suggested but no luck.

I do have the other four devices to update if I ever do get A11 working and that seems to be a pretty long process compared to the ready built 8.1 ROM, so I need to consider that too.

Not sure what else to try. These are positioned around the house running my nice dashboard so need to be fairly robust if possible. Fully Kiosk is great for managing and I have a system management dashboard setup where I can set top and side bar on/off and reload the dashboards etc.

I guess it would be great to have an A11 ROM packaged like mattmon has done to make it a bit easier to install.

Thanks :)

**Comment by dinki** [comment]
Yep.  The device is both fantastic and terrible all at the same time.  I had a lot of frustration getting where I am.  I received my second device and hope to put up a guide on flashing it but who knows if things will go smoother even knowing what I know now.  I went from the A8 to A11 back to A8 then finally back to A11.  I was in a mode of testing things but did end up with A11 where I am now.   I did initially flash A8 using Linux with Matt's help but later went to the Windows method I provide in the wiki.  Of course all machines are different so what worked for me might not work for others.

In regards to your current Lenovo screen crash, you may try entering recovery and clearing the cache or do a factory reset if that doesn't work.  I know I went through something similar but I don't know for sure if that helped.

I've heard mention of an updated ROM but I can't remember if it was A8 or A11 coming.  My biggest issue with A8 is the lack of sound from the browser.  I can't remember if webview was also a problem but that's a bit of a dealbreaker for me as well.

Just rambling.

**Comment by pgale** [comment]
I'll give that a go and try some more over the weekend. There's also the PMOS development and apparently they've got quite far but I'd prefer Android really due to the benefits Fully Kiosk brings. Might have a look at it a bit more but not seen an install guide yet so could be another steep learning curve!

**Comment by dinki** [comment]
Don't know anything about PMOS.  Will check it out.

Side note, I've opened up discussions (see nav bar) to have a better place for things like this.
